server/src/main/java/com/cloud/network/element/VirtualRouterElement.java

```diff
@@ -702,18 +702,27 @@ public boolean savePassword(final Network network, final NicProfile nic, final V
 
         // If any router is running then send save password command otherwise
         // save the password in DB
+        boolean savePasswordResult = true;
+        boolean isVrRunning = false;
```

Comment:
@ravening can you please make both variables private?

Reply:
@GabrielBrascher isnot variables private by default ?

Reply:
yes @weizhouapache @GabrielBrascher , and i don't even think the keyword is valid in that place. so unless you are asking to move them to fields...?

Reply:
@GabrielBrascher @DaanHoogland do you still want me to add private keyword?

Reply:
You are right, I did not expand correctly the diff and saw it as on the class instead of method.